sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -111,15 +111,15 @@ export interface AccessPolicy {
   /**
    * the permissions for the acl policy
    */
-  permission: string;
+  permissions: string;
 }
 
 /**
  * Properties of a blob
  */
 export interface BlobProperties {
-  creationTime?: Date;
-  lastModified: Date;
```

Comment:
>lastModified [](start = 2, length = 12)

`lastModified` is fine.  The comments only says renaming those with `Time` prefix/suffix

Reply:
Let me check with other languages. If they are not changing this one we won't either.

---
In reply to: [335205316](https://github.com/Azure/azure-sdk-for-js/pull/5580#discussion_r335205316) [](ancestors = 335205316)

Reply:
Reverted to lastModified